source/lmp/pair_deepmd.cpp

```diff
@@ -1059,7 +1105,7 @@ void PairDeepMD::settings(int narg, char **arg) {
   if (do_ttm && aparam.size() > 0) {
     error->all(FLERR, "aparam and ttm should NOT be set simultaneously");
```

Comment:
`do_compute_aparam` should be added
```suggestion
  if ((int)do_ttm + (int)do_compute_aparam + (int)(aparam.size() > 0) > 1) {
    error->all(FLERR, "aparam, aparam_from_compute, and ttm should NOT be set simultaneously");
```